### DISCUSSION TITLE
Can constraints be enforced between RCareWorld objects?

### DISCUSSION THREAD

**Discussion by nikkijhart** [empriselab/RCareWorld#13] [Help - Programming] [answered]
Can constraints be enforced between RCareWorld objects like they can in pybullet? I'm not sure if the human/wheelchair are registered as objects in pybullet and I can therefore use the pybullet createConstraint method, or if there is a better way using the RCareWorld API

**Comment by YoruCathy** [comment]
If you want the human and wheelchair to move together you can use setParent to set them as parent/child. Other than that you can use articulation units to make revolute/prismatic/other kinds of joints. The best practice would be using Unity's constraints: https://docs.unity3d.com/Manual/Constraints.html

**Reply by nikkijhart** [reply]
Is there a way to do this dynamically in the code, or can it only be set in a scene in Unity?

**Reply by YoruCathy** [reply, marked as answer]
Check this https://docs.unity3d.com/Manual/Joints.html
